BackEnd/src/main/java/MusicAPI/harmonicsKB/MelodicMotion.java

```diff
@@ -0,0 +1,15 @@
+package MusicAPI.harmonicsKB;
+
+/**
+ * Created by ben on 2/7/2016.
+ */
+public enum MelodicMotion {
```

Comment:
All these objects should be in a separate folder probably to making understanding easier